AWS with Terraform module (#10)
secrethub/secrethub-kubernetes-mutating-webhook: deploy/aws-lambda/README.md

# Deploy to AWS Lambda

You can deploy the webhook to AWS Lambda and activate it in your Kubernetes cluster by adding the following module to your Terraform project: 

```terraform
module "secrethub_mutating_webhook" {

>   source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook?ref=v0.2.0/deploy/aws-lambda"

Reviewer: Does this work? 🤔
First the `ref` query parameter and then the local path?

Reviewer: I'd expect:
```suggestion
  source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook//deploy/aws-lambda?ref=v0.2.0"
```
(didn't test this)

Author: Yes, indeed it works just like it is presented.
The one I tried was the following and it didn't work:
`github.com/secrethub/secrethub-kubernetes-mutating-webhook/deploy/aws-lambda?ref=v0.2.0`
Therefore I found the one that is implemented.

Interesting enough, your suggestion works, which I think it is indeed cleaner. I will apply your version.

Author: Fixed.

> ## Deploy in a AWS Lambda Function

Reviewer: ```suggestion
# Deploy to AWS Lambda
```

> You can deploy the webhook to an AWS Lambda function and activate it in your Kubernetes cluster by adding the following module to your Terraform file: 

Reviewer: ```suggestion
You can deploy the webhook to AWS Lambda and activate it in your Kubernetes cluster by adding the following module to your Terraform project: 
```

```terraform
module "secrethub_mutating_webhook" {

>   source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook/deploy/aws-lambda"

Reviewer: We should version this. I think you can do this by appending `?ref=v0.2.0`.

Author: Added the version

}

> ```

Reviewer: We should mention that it depends on the AWS and Kubernetes providers:

````suggestion
```

This module requires the [AWS provider](https://registry.terraform.io/providers/hashicorp/aws/latest/docs) and [Kubernetes provider](https://registry.terraform.io/providers/hashicorp/kubernetes/latest/docs) to be configured in your root project.
````
